README.md

```diff
@@ -1,6 +1,9 @@
 # OpenTelemetry .NET
 [![Gitter chat](https://img.shields.io/gitter/room/opentelemetry/opentelemetry-dotnet)](https://gitter.im/open-telemetry/opentelemetry-dotnet)
-[![Build Status](https://dev.azure.com/opentelemetry/pipelines/_apis/build/status/open-telemetry.opentelemetry-dotnet-myget-update?branchName=master)](https://dev.azure.com/opentelemetry/pipelines/_build/latest?definitionId=2&branchName=master)
+
+[![Build Status](https://github.com/open-telemetry/opentelemetry-dotnet/workflows/.NET%20Linux/badge.svg?branch=master)]
+[![Build Status](https://github.com/open-telemetry/opentelemetry-dotnet/workflows/.NET%20Windows/badge.svg?branch=master)]

```

Comment:
Check the [rendered result](https://github.com/cijothomas/opentelemetry-dotnet/blob/cijothomas/readmebadge/README.md), the brackets seem to be redundant.

![image](https://user-images.githubusercontent.com/17327289/86727554-623d8580-bfe0-11ea-84c4-02ea3cad145d.png)